README.md

```diff
@@ -40,6 +40,7 @@ We encourage pull requests and issues tracking via Github, and the [target-devel
    * *Note:* If using systemd, `-DSUPPORT_SYSTEMD=ON -DCMAKE_INSTALL_PREFIX=/usr` should be passed to cmake, so files are installed to the correct location.
 1. Type `make`
 1. Type `make install`
+1. Type `xargs rm < install_manifest.txt` to uninstall from source
```

Comment:
Is `make uninstall` not functional?

Reply:
@nixpanic
Yeah, for the cmake this is not functional as default, please see [1].

[1] https://gitlab.kitware.com/cmake/community/wikis/FAQ#can-i-do-make-uninstall-with-cmake

Or just add one as the wiki does ? Not sure whether will it make sense here by make it more complicated.

Thanks.